osm2gtfs/core/stops.py

```diff
@@ -0,0 +1,92 @@
+# coding=utf-8
+
+import sys
+import attr
+
+
+@attr.s
+class Station(object):
+
+    osm_id = attr.ib()
+    tags = attr.ib()
+    lat = attr.ib()
+    lon = attr.ib()
+    name = attr.ib()
+
+    stop_id = attr.ib(default="")
+    osm_type = attr.ib(default="relation")
+    location_type = attr.ib(default=1)
+    osm_url = attr.ib(default="http://osm.org/" +
+                      str(osm_type) + "/" + str(osm_id))
```

Comment:
As in all the other cases, shouldn't we leave that one in `__attrs_post_init__` (here duplicated). Plus: Is there any reason you never use `https://` instead of `http://`?

Reply:
Yes, I will change.

The reason for using `http` instead of `https` is, because when we started using the shorter link, the redirection was only working for `http`, but I checked and it seemed to be improved on OpenStreetMap.org, so we can perfectly switch here.